src/main/resources/META-INF/csrfguard-jahia.properties

```diff
@@ -109,7 +109,7 @@ org.owasp.csrfguard.JavascriptServlet.sourceFile = META-INF/csrfguard.template.j
 
 # If TokenPerPage is false, the javascript should currently rather not be cached on the browser
 # see discussion: https://github.com/OWASP/www-project-csrfguard/discussions/57
-org.owasp.csrfguard.JavascriptServlet.cacheControl = no-cache, no-store
+org.owasp.csrfguard.JavascriptServlet.cacheControl = private, max-age=28800
```

Comment:
The abvove 4 lines can be deleted now, as the setting is now again the same as in the original file, so our added comment above is now misleading. The setting could theoretically even be removed from the file or commented out, as the setting will be taken from the original config file.

Also the comment above on line 96 has recently (for version 4.4.0) changed in the original file saying that the default now is META-INF/csrfguard.min.js . Basically csrfguard has also now added the feature to minify and transpile the js: https://github.com/OWASP/www-project-csrfguard/issues/31 We added our own way of doing it before them, so maybe it would be worth to merge it now and do it the same way as they do.